### ISSUE TITLE
Figure out a way to calculate feature importances for big models

### ISSUE THREAD

**Issue by marco-c** [mozilla/bugbug#419]
Right now, calculating importances for e.g. the component model or the defect/enhancement/task model runs out of memory.

**Comment by marco-c** [comment]
The out of memory is happening at https://github.com/mozilla/bugbug/blob/0bf7234615a01f3b20a8a01ef76fc8b6521f90a3/bugbug/model.py#L121.

Fixing #162 might fix this too.

**Comment by chidauri** [comment]
@marco-c  figured out, it is actually happening at these lines for big models.
https://github.com/mozilla/bugbug/blob/ed8da961f11ca2a9345b54e9e5626d72d7880c8f/bugbug/model.py#L192-L193

**Comment by marco-c** [comment]
@chidauri yes, the other one I mentioned is a separate OOM, but we can ignore that one as it's going to automatically be fixed.

**Comment by marco-c** [comment]
https://github.com/slundberg/shap/issues/694

**Comment by marco-c** [comment]
A possible option might be to look at the `kmeans` function in shap.
